backend/grpc.go

```diff
@@ -0,0 +1,183 @@
+package backend
```

Comment:
Can you add copyright to all the new files.

Reply:
Done.